user_docs/en/userGuide.t2t

```diff
@@ -2591,7 +2591,7 @@ To access the Add-on Store from anywhere, assign a custom gesture using the [Inp
 
 ++ Browsing add-ons ++[AddonStoreBrowsing]
 When opened, the Add-on Store displays a list of add-ons.
-You can jump back to the list with ``alt+l`` from anywhere else within the store.
+You can jump back to the add-ons list with ``alt+a`` from anywhere else within the store.

```

Comment:
I agree, this line can be removed. There's also a reference to ``alt+a`` for actions which should be removed or updated. Additionally, you may need to update references in `tests\manual\addonStore.md`

```suggestion
```